tools/chocolateyinstall.ps1

```diff
@@ -4,22 +4,35 @@ $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 Get-ChildItem -Path "$toolsDir\tailscale.cer" | Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher
 
-$url = 'https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.34.1.exe'
-
-
 $packageArgs = @{
-  packageName    = $env:ChocolateyPackageName
-  unzipLocation  = $toolsDir
-  fileType       = 'exe'
-  url            = $url
-
-  softwareName   = 'Tailscale'
-
-  checksum       = '302b5ef05ad97d39c81624d77882b67adfef2e68da4c9e2a9ebbf5667c29a3b0'
-  checksumType   = 'sha256'
+  SoftwareName   = 'Tailscale'
+  PackageName    = $env:ChocolateyPackageName
+  UnzipLocation  = $toolsDir
+  FileType       = 'msi'
 
-  silentArgs     = '/S'
+  silentArgs     = '/quiet'
   validExitCodes = @(0)
 }
 
+if ($env:PROCESSOR_ARCHITECTURE -eq 'ARM64') {
```

Comment:
```suggestion
if ($env:PROCESSOR_IDENTIFIER.StartsWith('ARMv')) {
```

This seems to be sufficient to make arm64 detection work for now. It's not perfect but maybe Chocolatey will improve that over time. Unfortunately, the msi itself seems to be failing with an error about a processor mismatch (this happens outside of chocolatey too) so I guess there's some more work to be done. The arm64 MSI did work before, so that's a recent problem.